cpp/src/ray/config_internal.cc

```diff
@@ -9,8 +9,8 @@ DEFINE_string(ray_redis_password, "",
               "Prevents external clients without the password from connecting to Redis "
               "if provided.");
 
-DEFINE_string(ray_dynamic_library_path, "",
-              "The local path of application's dynamic library.");
+DEFINE_string(ray_code_search_path, "",
+              "The search path of application's dynamic libraries.");
```

Comment:
Could you copy the comment from `gcs.proto`?

Reply:
```
// A list of directories or jar files that specify the search path for user
  // code. This will be used as `CLASSPATH` in Java, and `PYTHONPATH` in
  // Python.
```
The description in gcs.proto is for all languages.

Reply:
You can remove the Java and Python parts.

Reply:
Maybe you can show the description in your mind?

Reply:
A list of directories or files of dynamic libraries that specify the search path for user code. Libraries under these paths will be loaded by 'dlopen'.